### DISCUSSION TITLE
Sollten Laufzettel besser zugänglich sein?

### DISCUSSION THREAD

**Discussion by oliver-stoehr** [kitodo/kitodo-production#5768] [General]
Aktuell können Laufzettel nur in der Vorgangsliste erstellt werden. Da nicht alle User Zugriff auf die Vorgangsliste haben, ist bei mir die Frage aufgekommen:  
**Sollten Laufzettel auch an anderen Stellen im System erstellt werden können?**

Eine möglicherweise sinnvolle Stelle dafür wäre die Aufgaben-Detailseite. Ob der Button zum Generieren von Laufzetteln an dieser Stelle angezeigt wird, könnte dafür im Workflow-Editor konfiguriert werden.

**Gibt es andere Stellen an denen der Button besser platziert wäre oder gibt es Gründe dafür, dass nur User mit Zugriff auf die Vorgangsliste Laufzettel erstellen können sollten?**

@kitodo/kitodo-community-board

**Comment by solth** [comment]
Ich denke auch, dass die Möglichkeit, Laufzettel zu drucken, nicht auf die Vorgangsliste beschränkt sein sollte. @subhhwendt, @andre-hohmann: wie ist eure Meinung hierzu? Wäre es sinnvoll, einen Button für die Funktion z.B. auf der Aufgabendetailseite einzufügen, damit der Laufzettel leichter gedruckt werden kann?

**Comment by subhhwendt** [comment]
Wir haben den Vorschlag gestern im Community Board besprochen und tragen folgende gemeinsame Haltung vor:

- eine Beschränkung dieses Features auf die Vorgangsliste ist nicht Ausdruck einer gewollten Beschränkung auf einen bestimmten Berechtiungsradius oder Funktionsumfang je nach Liste
- umgekehrt gilt aber auch, dass im praktischen Leben die Personen mit der Aufgabe zur Vorgangserstellung die Berechtigung für die Vorgangsliste haben werden und dass das Anlegen von neuen Vorgängen mit einem geschmeidigen Umschalten in die Vorgangsliste endet, wo dann der neue Vorgang als erster Eintrag in der Liste zum Ausdrucken eines Laufzettels anzutreffen ist.
- Die vorgeschlagene Erweiterung wird zumindest in diesem Kreis nicht vermisst.

**Reply by solth** [reply]
@subhhwendt Danke für die Zusammenfassung. Bedeutet dies im Umkehrschluss aber auch, dass eine entsprechende Erweiterung nicht als störend oder gar schädlich empfunden wird und damit einer Implementierung konzeptionell nichts im Weg steht? Soweit ich weiß, gibt es ein paar Einrichtungen, denen diese Erweiterung die Verwendung von Kitodo.Production vereinfachen würde.

**Comment by subhhwendt** [comment]
Hier kommt noch das fehlende Statement - sorry ...
Einer Implementierung des Laufzettes in der Aufgabenliste steht aus unserer Sicht nichts entgegen! Da die Abläufe in den Einrichtungen in den Feinheiten dann doch zuweilen unterschiedlich sind, wollen wir keinesfalls verhindern, dass der Workflow bei anderen geschmeidiger wird ;-)
